packages/react-catalog-view-extension/src/components/VerticalTabs/VerticalTabsTab.tsx

```diff
@@ -29,14 +35,19 @@ export const VerticalTabsTab: React.FunctionComponent<VerticalTabsTabProps> = ({
   hasActiveDescendant = false,
   shown = false,
   onActivate = null,
+  href,
+  isExternalLink,
   ...props
 }: VerticalTabsTabProps) => {
   const classes = css('vertical-tabs-pf-tab', { active, 'active-descendant': hasActiveDescendant, shown }, className);
 
-  const linkClasses = css({
-    'no-wrap': wrapStyle === 'nowrap',
-    truncate: wrapStyle === 'truncate'
-  });
+  const linkClasses = css(
+    {
+      'no-wrap': wrapStyle === 'nowrap',
+      truncate: wrapStyle === 'truncate'
+    },
+    href && textUtilities.linkColor
```

Comment:
This update should probably be made in the extension CSS. Our utilities add `!important` to all of the rules, which is a pretty bad practice for regular component styles - those are for users to use if they prefer styling that way.

From the looks of it, maybe this could be solved with attribute selectors? Do these examples help - https://codepen.io/mcoker/pen/wvdmVxB

Reply:
I dropped the idea of enforcing styling for now. See comment below